File: src/main/java/no/nav/pensjon/regler/domain/beregning2011/IFremskriving.java
package no.nav.pensjon.regler.domain.beregning2011;

public interface IFremskriving extends IJustering {

    public abstract double getTeller();

    public abstract void setTeller(double teller);

    public abstract double getNevner();

    public abstract void setNevner(double nevner);
}

File: src/main/kotlin/no/nav/pensjon/regler/domain/beregning2011/IFremskriving.kt
package no.nav.pensjon.regler.domain.beregning2011

import com.fasterxml.jackson.annotation.JsonSubTypes
import com.fasterxml.jackson.annotation.JsonTypeInfo

@JsonSubTypes(
    JsonSubTypes.Type(value = FremskrivingsDetaljer::class)
)
@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "type")
interface IFremskriving : IJustering {
    var teller: Double
    var nevner: Double
}